movie_flow_create.sql

```diff
@@ -0,0 +1,77 @@
+CREATE SCHEMA IF NOT EXISTS movie_flow;
+SET search_path = movie_flow;
+
+CREATE TABLE IF NOT EXISTS movie (
+    id SERIAL PRIMARY KEY,
+    release_date TIMESTAMP NOT NULL,
```

Comment:
You have TIMESTAMP datatype here but in data insertion script you are inserting values like '2022-01-15', '2019-05-02' without specified time, so looks like we can change type to just DATE